arrows/ocv/motion_detector_MOG2.cxx

```diff
@@ -0,0 +1,281 @@
+/*ckwg +29
```

Comment:
Are we moving away from including the copyright in each file (per a recent kitware-wide email thread)?

Reply:
That has been discussed, and I would like to, but there is no action being taken as of now to make the change in KWIVER.  It requires a bit of work to update every file. Maybe this is a task for one of our software interns?

Reply:
Can we at least decide to not care about making sure the data in each individual copyright is correct since it will be removed altogether in the near future?

Reply:
I think that makes sense once we've made a firm commitment to make this change and, ideally, assigned someone to make it.  If not then the change will never happen and we'll just be in more of a mess.  @kfieldho is this something one of the software interns could tackle?